### DISCUSSION TITLE
Sudo installing from Jupyter notebook

### DISCUSSION THREAD

**Discussion by Brybrio** [sr320/course-fish546-2021#20] [Q&A] [answered]
I was trying to sudo install curl from the Jupyter notebook but it would not work after asking for my password. It was easily fixed by running it in the command line, but just curious on how to get it done from Jupyter

**Comment by kubu4** [comment, marked as answer]
Just documenting discussion from class today.

Jupyter notebooks don't provide any sort of "interactive" interface. Generally, any commands that would require a response from the user won't work.
